src/Forms/FixCommonErrors.cs

```diff
@@ -4802,10 +4799,13 @@ private void ToolStripMenuItemDeleteClick(object sender, EventArgs e)
                 Next();
 
                 // restore de-selected fixes
-                foreach (ListViewItem item in listViewFixes.Items)
+                if (deSelectedFixes.Count > 0)

```

Comment:
Can be *0* if `listViewFixes.SelectedItem` was included in deletion, so there is no need to run the *0(n)*